ministryofjustice/correspondence_tool_staff
# CT-2830: Minor wording

## Description
Minor wording changes relate to the page for moving and join business 
 
## Self-review checklist

 
### Screenshots

 after: (before please check the Jira ticket)
![Screenshot 2020-05-13 at 11 05 20](https://user-images.githubusercontent.com/26111671/81801876-fe7d6b80-950c-11ea-95e5-1fc616fb743b.png)

![Screenshot 2020-05-13 at 11 05 25](https://user-images.githubusercontent.com/26111671/81801921-11903b80-950d-11ea-8f70-4ae3be9a0aff.png)

### Related JIRA tickets

 https://dsdmoj.atlassian.net/secure/RapidBoard.jspa?rapidView=25&projectKey=CT&modal=detail&selectedIssue=CT-2830

### Deployment

 No need to do anything 

### Manual testing instructions

- Login in with David account if you know
- Choose "Team" from menu and choose one of business group, then continue to go down to a business unit level
- Click "Move the ....",  check the wording
- Come back to previous page and Click "Join...", check the wording
